docs/community.rst

```diff
@@ -8,13 +8,13 @@ who can answer your questions and help you out. Please do join us.
 The two major places for community discussion are:
 
 * `The mailing list <https://groups.google.com/forum/#!forum/hypothesis-users>`_.
-* An IRC channel: #hypothesis on freenode.
+* An IRC channel: #hypothesis on Freenode.
 
 Feel free to use these to ask for help, provide feedback, or discuss anything remotely
 Hypothesis related at all.
 
 The IRC channel is the more active of the two. If you don't know how to use
-IRC, don't worry about it. Just `click here to sign up to IRCCloud and log in <https://www.irccloud.com/invite?hostname=irc.freenode.net&channel=%23hypothesis>`_
+IRC, don't worry about it. Just `click here to sign up to IRCCloud and log in <https://www.irccloud.com/invite?hostname=irc.Freenode.net&channel=%23hypothesis>`_
```

Comment:
It's a small thing, because this text isn't user-facing, but generally URLs are lowercase. Seeing uppercase in URLs like this is a bit unusual.